src/components/Login.tsx

```diff
@@ -0,0 +1,130 @@
+"use client";
+
+import { useState } from "react";
+import { Button } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import Link from "next/link";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form";
+import { z } from "zod";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import { Eye, EyeOff, Lock, Mail } from "lucide-react";
+
+const formSchema = z.object({
+  email: z.string()
```

Comment:
We will not authenticating user against email. Try using phone number. Try using input masking of `0321-1234567`

Reply:
**Fixed**